### PR TITLE
Asset compute generator

### PR DESCRIPTION
**Pull request by purplecabbage**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: generators/add-action/asset-compute/templates/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (tmathern) [inline generators/add-action/asset-compute/templates/.eslintrc.json:1]: We actually decided to totally remove the eslint config. If you could remove it there too...

File: lib/constants.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ module.exports = {
   manifestPackagePlaceholder: '__APP_PACKAGE__',
   sdkCodes: {
     analytics: 'AdobeAnalyticsSDK',
-    assetCompute: 'AssetCompute',
+    assetCompute: 'AssetComputeSDK',
     campaign: 'CampaignSDK',
     customerProfile: 'McDataServicesSdk',
```
Comment (tmathern) [inline lib/constants.js:10]: `customerProfile` should also have its SDK in uppercase, I think (consistency).

**Review by tmathern — CHANGES_REQUESTED** [review]
See inline. Also, https://github.com/adobe/generator-aio-app/pull/55 on which I'm working also already has some of those changes.

File: lib/constants.js (hunk 2 above)
Comment (purplecabbage) [reply lib/constants.js:10]: we cannot change product codes ... yes it should be consistent, but it is not

**Comment by purplecabbage** [comment]
I was trying to help, if you just want to make these changes in #55 yourself please go ahead.

**Comment by tmathern** [comment]
@purplecabbage It's fine :) @jdelbick should review too, and merge (it seems I can't).

**Pull request closed without merging**